src/app/pages/LiquidityMining/components/HistoryTable/index.tsx

```diff
@@ -1,83 +1,61 @@
-import React, { useEffect, useState, useCallback, useRef } from 'react';
-import axios, { CancelTokenSource } from 'axios';
+import React, { useState, useCallback, useMemo } from 'react';
 
-import { backendUrl, currentChainId } from 'utils/classifiers';
 import { useAccount } from 'app/hooks/useAccount';
 import { TableBody } from './TableBody';
 import { TableHeader } from './TableHeader';
 import { Pagination } from 'app/components/Pagination';
-import { LiquidityMiningEvent, LiquidityMiningHistory } from './types';
+import { useGetLiquidityHistoryQuery } from 'utils/graphql/rsk/generated';
+import { LiquidityPoolDictionary } from 'utils/dictionaries/liquidity-pool-dictionary';
 
 const pageSize = 6;
 
 export const HistoryTable: React.FC = () => {
   const account = useAccount();
-  const url = backendUrl[currentChainId];
-  const [history, setHistory] = useState<LiquidityMiningEvent[]>([]);
-  const [loading, setLoading] = useState(false);
   const [page, setPage] = useState(1);
-  const [total, setTotal] = useState(0);
-
-  let cancelTokenSource = useRef<CancelTokenSource>();
-  const getData = useCallback(() => {
-    if (cancelTokenSource.current) {
-      cancelTokenSource.current.cancel();
-    }
-
-    cancelTokenSource.current = axios.CancelToken.source();
-    axios
-      .get<LiquidityMiningHistory>(
-        `${url}/liquidity-history/${account}?page=${page}&pageSize=${pageSize}`,
-        {
-          cancelToken: cancelTokenSource.current.token,
-        },
-      )
-      .then(res => {
-        const { events, pagination } = res.data;
-        setHistory(events);
-        setTotal(pagination.totalPages * pageSize);
-        setLoading(false);
-      })
-      .catch(e => {
-        console.log(e);
-        setHistory([]);
-        setLoading(false);
-      });
-  }, [url, account, page]);
-
-  const getHistory = useCallback(() => {
-    setLoading(true);
-    setHistory([]);
-
-    getData();
-  }, [getData]);
-
-  useEffect(() => {
-    if (account) {
-      getHistory();
-    }
-  }, [account, getHistory]);
+  const { data, loading } = useGetLiquidityHistoryQuery({
+    variables: {
+      user: account.toLowerCase(),
+    },
```

Comment:
```suggestion
    },
    pollInterval: APOLLO_POLL_INTERVAL,
```

this can be imported from `utils/classifiers.ts`